dist/spec.yaml

```diff
@@ -11824,6 +11864,38 @@ definitions:
         type: string
     x-okta-tags:
       - User
+  ThreatInsightConfiguration:
+    properties:
+      _links:
+        additionalProperties:
+          type: object
+        readOnly: true
+        type: object
+      action:
+        type: string
+      created:
+        format: date-time
+        readOnly: true
+        type: string
+      excludeZones:
+        items:
+          type: string
+        type: array
+      lastUpdated:
+        format: date-time
+        readOnly: true
+        type: string
+    x-okta-crud:
+      - alias: read
```

Comment:
We don't use `x-okta-crud` in .NET, but I'm not sure we use `read`/GET in other entities. Can you check with someone that utilizes the crud section? Maybe @bretterer ?

Reply:
This is waiting for @bretterer to review

Reply:
`x-okta-crud` section looks good to me. I had verified Java SDK code generation with it. cc: @sergiishamrai-okta